qiskit_cold_atom/circuit_tools.py

```diff
@@ -185,13 +225,71 @@ def circuit_to_cold_atom(
         # validate the circuits against the backend configuration
         cls.validate_circuits(circuits=circuits, backend=backend, shots=shots)
 
+        if "wire_order" in backend.configuration().to_dict():
+            wire_order = backend.configuration().wire_order
+        else:
+            wire_order = cls.__wire_order__
+
         experiments = {}
         for idx, circuit in enumerate(circuits):
             experiments["experiment_%i" % idx] = {
-                "instructions": cls.circuit_to_data(circuit),
+                "instructions": cls.circuit_to_data(circuit, backend=backend),
                 "shots": shots,
                 "num_wires": circuit.num_qubits,
-                "wire_order": cls.__wire_order__,
+                "wire_order": wire_order,
             }
 
         return experiments
+
+    @classmethod
+    def convert_wire_order(
+        cls,
+        wires: List[int],
+        convention_from: str,
+        convention_to: str,
+        num_sites: int,
+        num_species: int,
+        sort: Optional[bool] = False,
+    ) -> List[int]:
+        """
+        Converts a list of wire indices onto which a gate acts from one convention to another.
+        Possible conventions are "sequential", where the first num_sites wires denote the
+        first species, the second num_sites wires denote the second species etc., and "interleaved",
+        where the first num_species wires denote the first site, the second num_species wires denote
+        the second site etc.
+
+        Args:
+            wires: Wires onto which a gate acts, e.g. [3, 4, 7, 8].
+            convention_from: The convention in which "wires" is given.
+            convention_to: The convention into which to convert.
+            num_sites: The total number of sites.
+            num_species: The number of different atomic species.
+            sort: If true, the returned list of indices is sorted in ascending order.
+
+        Raises:
+            QiskitColdAtomError: If the convention to and from is not supported.
+
+        Returns:
+            A list of wire indices following the convention_to.
+        """
+        if (convention_to or convention_from) not in ["sequential", "interleaved"]:
```

Comment:
```suggestion
        if (convention_to or convention_from) not in WireOrder:
```
However, this check may no be needed anymore if we assume that instances of `WireOrder` are given.